Modules/_testcapimodule.c

```diff
@@ -1039,6 +1039,32 @@ test_buildvalue_N_error(const char *fmt)
     return 0;
 }
 
+static PyObject *
+test_buildvalue_p(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *res = Py_BuildValue("p",  3);
+    if (res == NULL) {
+        return NULL;
+    }
+    if (!Py_IsTrue(res)) {
```

Comment:
`Py_IsTrue()` can raise an exception. Also, this test is too lenient.
```suggestion
    if (res != Py_True) {
```

Reply:
I don't understand your concern about Py_IsTrue(). It's a thin wrapper for (res == Py_True).

Py_IsTrue() cannot raise an exception. Are you thinking at PyObject_IsTrue()?

Reply:
Ah, I was thinking about `PyObject_IsTrue()`. It is confusing. Why not use just `Py_True`?

Reply:
I added the function for interoperability with PyPy & other Python implementations: https://bugs.python.org/issue43753

Using it in the CPython is to show others how to use it. In CPython, Py_IsTrue(x) and (x == Py_True) is exactly the same.